### PR TITLE
Improve drill settings

### PR DESCRIPTION
**Pull request by hackerstanislav**
JIRA: RAIL-4583

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/10228/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12203/ : SUCCESS in 12m 30s

File: libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/AttributeUrlSection.tsx [hunk 1]
```diff
@@ -51,23 +51,17 @@ export const AttributeUrlSection: React.FC<AttributeUrlSectionProps> = (props) =
...
-                                    () => {
-                                        /**/
-                                    }
-                                    // onClickHandler(
-                                    //     e,
-                                    //     item.ref,
-                                    //     item.ref, // TODO
-                                    // )
+                                onClickHandler={(e) =>
+                                    onClickHandler(e, target.displayForm, target.hyperlinkDisplayForm)
```
Comment (ivanmjartan) [inline libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/AttributeUrlSection.tsx:64]: Remove lamda here and decompose  each attribute in onClickHandler = useCallback( ...  than you will have memoized callback ;-)

**Review by ivanmjartan — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline gate aborted.

**Review by ivanmjartan — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3109

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2944/ : FAILURE in 1m 37s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3250/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2096/ : SUCCESS in 21m 19s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/12206/ : SUCCESS in 12m 41s

**Pull request merged**